app/components/views/TicketsPage/MyTicketsTab/MyTicketsTab.jsx

```diff
@@ -15,6 +17,13 @@ const labels = {
   [txTypes.REVOKED]: <T id="ticket.status.multiple.revoked" m="revoked" />
 };
 
+const qrHashes = [];
```

Comment:
this logic should move to useTicketsList custom hook, which should expose minimal set of variables to the components to use.